revel.go

```diff
@@ -16,6 +16,8 @@ import (
 
 const (
 	REVEL_IMPORT_PATH = "github.com/revel/revel"
+	DEFAULT_DATE_FORMAT     = "2006-01-02"
+	DEFAULT_DATETIME_FORMAT = "2006-01-02 15:04"
```

Comment:
@brendensoares Have you decided to restore old time format (#878)? Then we should make changes to config's template as well (https://github.com/revel/revel/blob/master/skeleton/conf/app.conf.template#L69). Because that's kind of counter intuitive to have different built in and config defaults.